grant/projects/offchain-subnames.md

```diff
@@ -0,0 +1,138 @@
+# Offchain Subnames on Darwinia Chain
+
+## Project Description
+
+This project aims to implement an offchain subnames solution for the 'ringdao.eth' Ethereum Name Service (ENS) domain but keeps the subname records on Darwinia Chain. 
```

Comment:
It should be ''darwinia.eth" instead of "ringdao.eth"

Reply:
done